collections/made-in-india/index.md

```diff
@@ -20,12 +21,11 @@ items:
  - SigNoz/signoz
  - uvdesk/community-skeleton
  - CRED-CLUB/synth-android
- - kadalu/kadalu
  - krishdevdb/reseter.css
- - project-sunbird
  - pupilfirst/pupilfirst
- - appsmithorg/appsmith
- - anuraghazra/github-readme-stats
+ - kadalu/kadalu
+ - TheAlgorithms
+ - project-sunbird
```

Comment:
Let's not put the orgs at the bottom but leave them where they were, thanks!

Reply:
@MikeMcQuaid Now I have placed the orgs nearby where the were. Like TheAlgorithms was in the top 3, and its most popular repo has >100k stars. Thus, Its on the top. And project-sunbird was in the last 3 and its most popular repo has ~150 stars being the least in the list. Thus, Its on the bottom